root/meta/rootmap/duplicateWarning/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+configure_file(rfile1.rootmap . COPYONLY)
+configure_file(rfile2.rootmap . COPYONLY)
+
+ROOTTEST_ADD_TEST(myclassduplicated
+                  COMMAND root -b -q -l
```

Comment:
This doesn't work on Windows, as remarked by Jenkins both here and in https://github.com/root-project/root/pull/13478#issuecomment-1686158862...